x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/installation_and_upgrade.md

```diff
@@ -978,14 +978,13 @@ Then the preview should display the changes for the newly selected rule
 
 #### **Scenario: User can see changes when updated rule is a different rule type**
 
-**Automation**: 1 UI integration test
+**Automation**: 1 e2e test
 
 ```Gherkin
 Given a prebuilt rule is installed in Kibana
 And this rule has an update available that changes the rule type
 When user opens the upgrade preview
 Then the rule type changes should be displayed in grouped field diffs with corresponding query fields
```

Comment:
Did I get it right the tooltip isn't implement yet?

Reply:
Yeah, we are pushing it back since we're not sure how best to implement it yet. I figured the best move would be to take it off here for now, but maybe we leave it in and just make a note to rewrite the test whenever it becomes available? What do you think?

Reply:
In my opinion a note in the test plan should convey the intention much better.